ftl/ftl_node_integration_tests.yaml

```diff
@@ -1,7 +1,7 @@
 steps:
 #Build node ftl image with bazel
   - name:  'gcr.io/cloud-builders/bazel'
```

Comment:
What's the issue here? That doing "bazel run" on this image tries to load it *and* run it, which errors? Run should just do the "load", but if it's actually doing a run as well I think we can handle that here by setting `legacy_run_behavior` to True:
https://github.com/bazelbuild/rules_docker/blob/master/container/image.bzl#L317

Reply:
Yah, so the "bazel run" does seem to be loading and running the image, I'll try using legacy_run_behavior to fix it!